.github/workflows/secrets-to-confidential.yml

```
@@ -0,0 +1,10 @@
+uses: jpoehnelt/secrets-sync-action@7840777f242539d96b60477b66aa1c179e7644ea #v1.10.0
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 1: did not find expected key

(syntax-check)